File: uri 1176 - Fibonacci Array.py
li = [0,1]
for i in range(2,61):
    li.append(li[i-1] + li[i-2])
t = int(input())
for i in range(t):
    n = int(input())
    print("Fib(%d) = %d"%(n, li[n]))

File: uri 1176 - Fibonacci Array.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    long long int a,i;
    int n;
    scanf("%d",&n);
    while(n>0){
        n--;
        scanf("%lld",&a);
        long long int d[a+1];
        for(i=0;i<=a;i++){
            if(i<=1){
                d[i]=i;
            }
            else{
                d[i]= d[i-1]+d[i-2];
            }
        }
        printf("Fib(%lld) = %lld\n",a,d[a]);
    }

    return 0;
}
